client/src/components/Verejne/entityHelpers.js

```diff
@@ -0,0 +1,16 @@
+// Entity which is a number is individual.
```

Comment:
I assume you did not touch this file, but if we're using it, please fix and add flow